### DISCUSSION TITLE
Appcopier 0.30

### DISCUSSION THREAD

**Discussion by Belim** [builtbybel/Appcopier#9] [Announcements]
**Here is a stable release of Appcopier. I have also prepared it as a setup package.** Source code is also pushed.
Some of you are now familiar with the Windows Backup feature in the Settings app or the Windows Backup app, whose UI does not allow for the restoration of specific settings and apps. Instead, you can only access your files through OneDrive, and for everything else, you only have the option to restore it during the initial setup.
With this release, Appcopier has taken another step forward and can now restore specific apps (internally handled through winget). Additionally, numerous UI corrections and bugs have been fixed. The app now weighs a few more KBs but remains lightweight. 

## Check out the new [MSBob app](https://github.com/builtbybel/MSBob)

<hr /><em>This discussion was created from the release <a href='https://github.com/builtbybel/Appcopier/releases/tag/0.30.0'>Appcopier 0.30</a>.</em>

**Comment by emorrank01** [comment]
Hi, Belim
I wonder if it is possible to add the system restore point feature.???

**Comment by geneh74** [comment]
Great job. It would be nice to have the option to specify where the backup will be saved to and where to restore from, so that I can backup to and restore from an external USB device, especially when C drive storage space is limited.
